api/tests/integration/slack/test_slack_environment_viewset.py

```diff
@@ -0,0 +1,177 @@
+import json
+
+import pytest
+from django.urls import reverse
+from rest_framework import status
+
+bot_token = "bot_token_test"
+
+
+@pytest.fixture
```

Comment:
Fixtures should live in a `conftest` module

Reply:
right, but they are only used in this module. Should we still move them?

Reply:
Yes, I think so.